tests/to-absolute-test.toit

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2023 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/TESTS_LICENSE file.
+
+import expect show *
+import fs
+import fs.windows
+import system
+import host.directory
+
+test-posix:
+  expect-equals directory.cwd (fs.to-absolute "")
+  expect-equals "/" (fs.to-absolute "/")
+  expect-equals "/a/b/c" (fs.to-absolute "/a/b/c")
+  expect-equals "/a/b/c" (fs.to-absolute "/a/b/c/") // clean removed the trainling slash
+  expect-equals "$directory.cwd/a/b/c" (fs.to-absolute "a/b/c")
+
+test-windows:
+  expect-equals directory.cwd (fs.to-absolute "")
+  expect-equals "$directory.cwd\\a" (fs.to-absolute "a")
+  expect-equals "c:\\a\\b" (fs.to-absolute "c:\\a\\b")
+  expect-equals "x:\\" (fs.to-absolute "x:\\")
+
+  if windows.is-absolute-volume directory.cwd:
+    // It seems impossible to fake UNC paths reliably on windows, so to test this probably, test
```

Comment:
Not sure if it works, but maybe chdir to `\\?\C:\foo\bar`  works? In that case you have something that resembles a UNC path. (replace "C:\foo\bar" with the actual cwd).

Reply:
It doesnt work. (I would think it is a bug in the primitive_file_windows.cc, it sill uses wstat64 that has some serious limitations). And `\\?\C:\foo\bar` is not even technically a UNC path.